packages/framework/src/Model/Navigation/NavigationItemFacade.php

```diff
@@ -119,15 +127,18 @@ protected function fixUrlInNavigationItemData(NavigationItemData $navigationItem
             return;
         }
 
-        if (strpos($navigationItemData->url, 'http') === 0) {
+        $domainUrl = $this->domain->getDomainConfigById($navigationItemData->domainId)->getUrl();
+        $navigationItemData->url = str_replace($domainUrl, '', $navigationItemData->url);
+
+        if (str_starts_with($navigationItemData->url, 'http')) {
             return;
         }
 
-        if (strpos($navigationItemData->url, 'www') === 0) {
+        if (str_starts_with($navigationItemData->url, 'www')) {
```

Comment:
about http and www check - what if I create and use a category named `WWW Tools and libraries`?

Reply:
changed the check to `www.` :slightly_smiling_face: